client/extensions/woocommerce/components/compact-tinymce/style.scss

```diff
@@ -0,0 +1,78 @@
+.compact-tinymce {
+	.mce-toolbar .mce-btn-group .mce-btn.mce-listbox:hover {
+		border-left-color: transparent;
+		border-right-color: lighten( $gray, 20% );
+	}
+
+	.mce-edit-area {
+		border: 1px solid lighten( $gray, 20% ) !important;
```

Comment:
instead of important is it preferable to do:

.mce-edit-area.mce-edit-area {
  border: 1px solid lighten( $gray, 20% )
}